pycsob/client.py

```diff
@@ -123,6 +124,7 @@ def payment_init(self, order_no, total_amount, return_url, description, merchant
             ('payMethod', 'card'),
             ('totalAmount', total_amount),
             ('currency', currency),
+            ('customer', utils.convert_keys_to_camel_case(customer_data)),
```

Comment:
We are now obliged to send customer data. https://github.com/csob/platebnibrana/wiki/Dodate%C4%8Dn%C3%A1-data-o-n%C3%A1kupu#customer-data-